suite-native/module-send/src/sendFormThunks.ts

```diff
@@ -87,6 +89,21 @@ export const cleanupSendFormThunk = createThunk(
     },
 );
 
+export const cleanupBitcoinSendFormsThunk = createThunk(
+    `${SEND_MODULE_PREFIX}/cleanupBitcoinSendFormsThunk`,
+    (_, { dispatch, getState }) => {
+        const sendFormDrafts = selectSendFormDrafts(getState());
+        const accountKeys = Object.keys(sendFormDrafts);
+
+        accountKeys.forEach(accountKey => {
+            const account = selectAccountByKey(getState(), accountKey);
+            if (account && hasNetworkFeatures(account, 'amount-unit')) {
+                dispatch(cleanupSendFormThunk({ accountKey }));
```

Comment:
```suggestion
                dispatch(sendFormActions.removeDraft({ accountKey }));
```

should be enough. Your implementation does not do any harm now, but might later if is the `cleanupSendFormThunk` modified or the send flow navigation between screens changed.

Reply:
OK, makes sense.